contracts/bridge/contracts/LightClientBridge.sol

```diff
@@ -158,37 +236,23 @@ contract LightClientBridge is Pausable, Initializable, ValidatorRegistry {
         uint256 validatorPosition,
         address validatorPublicKey,
         bytes32[] memory validatorPublicKeyMerkleProof
-    ) public payable whenNotPaused {
-        /**
-         * @dev Check if validatorPublicKeyMerkleProof is valid based on ValidatorRegistry merkle root
-         */
-        require(
-            checkValidatorInSet(
-                validatorPublicKey,
-                validatorPosition,
-                validatorPublicKeyMerkleProof
-            ),
-            "Error: Validator must be in validator set at correct position"
-        );
-
-        /**
-         * @dev Check if validatorSignature is correct, ie. check if it matches
-         * the signature of senderPublicKey on the commitmentHash
-         */
-        require(
-            ECDSA.recover(commitmentHash, validatorSignature) ==
-                validatorPublicKey,
-            "Error: Invalid Signature"
-        );
-
+    ) public payable {
```

Comment:
why it is payable?

Reply:
The `relayer` may lock up a stake for collateral, A `minisupport` is needed.
Before we clarify the `minisupport`, I will remove `payable` first.